pom.xml

```diff
@@ -239,7 +239,7 @@
         <jts.spatial4j.version>1.16.0</jts.spatial4j.version>
         <jvnet-ogc.version>2.6.1</jvnet-ogc.version>
         <jwnl.version>1.3.3</jwnl.version>
-        <karaf.version>4.2.9</karaf.version>
+        <karaf.version>4.3.0</karaf.version>
```

Comment:
There may be other dependencies that get upgraded with Karaf which should be updated to match the version that Karaf provides. For example, the Jetty version should be 9.4.31.v20200723 now (https://mvnrepository.com/artifact/org.ops4j.pax.web/pax-web-jetty/7.3.9). I would check whether Blueprint was upgraded too

Reply:
Sounds good, I wasn't sure if every dependency Karaf provides should be updated or just ones needed to get things working, but I can go through and update the rest